### ISSUE TITLE
[Masthead/Footer] Accessibility - Multiple elements with "banner" role do not have unique labels

### ISSUE THREAD

**Issue by SR1947** [carbon-design-system/carbon-for-ibm-dotcom#6783]
IBM Equal Access Accessibility Checker
Multiple elements with "banner" role do not have unique labels

<header class="bx--header">

Adopter:  SDMA - Passport Advantage

**Comment by RobertaJHahn** [comment]
@sr1947 This issue does not have enough information to resolve.  Please us the bug template and resubmit if this fix is still needed.  Ideally, including a screen shot of the error will help as well.  Closing this bug.
